Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1447,6 +1458,55 @@ def validate_long_running_params(params: Dict) -> None:
                                    ' Please set a value for it.')
 
 
+def get_cidrs_indicators(query):
+    """Extracts cidrs from a query"""
+    if not query:
+        return []
+
+    res = demisto.searchIndicators(query=query)
+
+    indicators = []
+    for indicator in res.get('iocs', []):
+        if indicator.get('indicator_type') == 'CIDR':
```

Comment:
Maybe make it case insensitive?
```suggestion
        if indicator.get('indicator_type').lower() == 'cidr':
```